### ISSUE TITLE
The result is different from your description! Help! Thanks!

### ISSUE THREAD

**Issue by zihao-lu** [soloice/mnist-bn#3]
when I set the parameter of istraining in batchnormalization as  False, I get low acuracy about 0.1, however, if i set it as true, the accuracy is 0.99+,why?

**Comment by soloice** [comment]
Sorry for my late reply.
It must be something weird with `tf.slim`. Replace it with `tf.layers.batch_normalization` will work.
You can check the new code.
